sdk/scheduler/src/main/java/com/mesosphere/sdk/config/DefaultConfigurationUpdater.java

```diff
@@ -271,8 +271,14 @@ private static boolean areDifferent(PodSpec podSpec1, PodSpec podSpec2) {
         }
 
         // Make counts equal, as only a difference in count should not effect an individual tasks.
-        podSpec1 = DefaultPodSpec.newBuilder(podSpec1).count(0).build();
-        podSpec2 = DefaultPodSpec.newBuilder(podSpec2).count(0).build();
+        podSpec1 = DefaultPodSpec.newBuilder(podSpec1).count(0).placementRule(null).build();
```

Comment:
Comments are spread out between lines for no reason.  Make one large block comment.